reference/performances_guide/index.rst

```diff
@@ -36,3 +36,17 @@ Here is the complete formula to check if you have, by far, more indexes than the
     + N enabled locales * N enabled channels (for the completeness filters)
     + 3 for family, groups and categories
     > 64
+
+Work with WYSIWYG :
+-------------------
+
+WYSIWYG are javascript component and so are computed client side. Hardware power of the client can affect the speed of pages with WYSIWYG.
+
+That's why it's not recommended to have more that one hundred WYSIWYG in the same page. In the product edit form, only attributes in the
+current attribute group and the current scope are rendered. It means, you should not have more than one hundred WYSIWYG in an attribute group
+and a scope. In mass edit and variant group, all scopes are rendered in the same time. It means you should not have more than one hundred
```

Comment:
"In mass edit and variant group" > unclear, do you mean "when performing mass edit or managinf variant groups" ?